subprojects/detekt.yml

```diff
@@ -620,7 +620,7 @@ style:
         ignoreNumbers: [ '-1', '0', '1', '2' ]
         ignoreHashCodeFunction: true
         ignorePropertyDeclaration: true
-        ignoreLocalVariableDeclaration: false
+        ignoreLocalVariableDeclaration: true
```

Comment:
What does it affect?

Reply:
extracting magic value to local variable didn't help with this flag disabled.

```
fun someFun() {
   someExpression(42) < -- MAGIC NUMBER
}
```

fix that didn't work before setting `ignoreLocalVariableDeclaration` to `true`:

```
fun someFun() {
   val meaningfulValueName = 42 < -- WAS MAGIC NUMBER STILL
   someExpression(meaningfulValueName) 
}
```